Documentation/Basics/GeneralConventions/GuidelinesForImages.rst

```diff
@@ -9,167 +9,58 @@
+*   If you need example data use :composer:`t3docs/site-package-data`
+*   As personalized usernames are considered best practice always use username
+    "kasper"
+*   Do not install third party extensions unless what you want to demonstrate
+    requires one. If possible use one of the extensions from vendor `typo3` or
+    `t3docs`.
+*   use PNG format (.png file ending)
+*   If you take a screenshot of a full page it should have 1400 x 1050 px
+*   Use only parts of a full page when possible, the flatter the screenshot the
+    less room it takes.
+*   When reviewing screenshots take into consideration that taking screenshots
+    is a lot of effort.
+
+.. _guidelines-for-images-screenshot-with-grafics:
 
 Guidelines for screenshots with graphics elements
 =================================================
 
 You will often see a screenshot where additional graphic elements have been added in the
-documentation. These additional graphic elements may be boxes, numbers or arrows. Up until,
-we did not have guidelines for this, so you may see various shapes in different colors.
-
-.. important::
-   This still needs some work. If you plan on adding images to a manual, contact
-   us in the #typo3-documentation channel. Help with optimizing this section is also
-   appreciated.
-
-
-.. index::
-   pair: Screenshots; Icons
-   pair: TYPO3; Icons
-
-To get the icons
-----------------
-
-Either download the zipfile and extract it.
-You can find the download link on https://github.com/TYPO3-Documentation/_Images,
-
-Or clone the repository:
-
-.. code-block:: bash
-
-   git clone https://github.com/TYPO3-Documentation/_Images
-
-
-The icons will be in images/final/icons.
-
-
-Follow these guidelines
------------------------
-
-*  Icons are used for numbers and arrows. The size of the icons should not be changed.
-*  Simple frames are TYPO3 Orange (#F49700) and have a line thickness of 2 px and a
-   corner radius of 2 px.
-*  Text fields are TYPO3 Orange (#F49700) and have a corner radius of 2 px.
-*  Dividing surfaces have a 2px frame in TYPO3 Orange (#F49700) with a corner radius
-   of 2 px.
-*  Icon shadows have an
-
-   * horizontal offset of 4 px
-   * vertical offset of 4 px
-   * blur radius of 10 px
-   * spread radius of 0 px
-   * color of RGBA (0, 0, 0, 0.3)
-
-
-Examples
---------
-
-
-.. figure:: images/backend-with-boxes.png
-   :class: with-shadow
-
-   PNG Image with rectangles and numbers
-
-.. figure:: images/backend-with-boxes-2.svg
-   :class: with-shadow
-
-   SVG image with rectangles and numbers
-
-.. figure:: /_Images/edit_me_on_github+shadow.svg
-   :class: with-shadow
-   :alt: Image with arrow
+documentation. These additional graphic elements may be boxes, numbers or arrows.
 
-   SVG Image with arrow
+*   Use sufficient contrast to ensure additional graphic elements are visible
+    across devices and for as many readers as possible, even if the do not see
+    a difference between red and green.
```

Comment:
There are more color vision deficiencies than red-green color blindness. Maybe we should make this more clear?

Reply:
Done